src/runtime.jl

```diff
@@ -72,6 +72,7 @@ function Base.wait(event::RuntimeEvent{HSAStatusSignal}; check_exceptions=true,
                 deleteat!(mod.metadata, idx)
             end
         end
+        deleteat!(active_kernels[get_default_queue()], findall(x->x==event, active_kernels[get_default_queue()]))
```

Comment:
We should probably get the queue from the event. I'm not sure if our `HSAStatusSignal` contains the queue it was launched on, so we'll possibly have to add that as a field.